website/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>druid-website</artifactId>
+  <name>druid-website</name>
+  <description>Website for druid.apache.org</description>
+
+  <parent>
+    <groupId>org.apache.druid</groupId>
+    <artifactId>druid</artifactId>
+    <version>27.0.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <!-- src repo of druid website, the artifacts of this build will be placed here -->
+    <website.src>../../druid-website-src</website.src>
+    <!--
+    'version' of website to build, by default it is the Druid version we are building, but can be set explicitly
+    to stage documentation for an unreleased version
+    -->
+    <website.version>${project.parent.version}</website.version>
+    <node.version>v16.17.0</node.version>
+    <npm.version>8.15.0</npm.version>
```

Comment:
are these the recommended docusaurus 2 versions?

Reply:
They don't have a recommended NPM version and the recommended node version is 16.14 or later